terraform/central-account/outputs.tf

```diff
@@ -13,16 +13,9 @@ If you're following the Terraform demo environment and your Terraform apply was
 successful, you can wait for a few minutes and then visit the ConsoleMe dashboard
 at the following URL:
 
-$ http://${module.server.public_ip[0]}:8081
+$ https://${aws_lb.public-to-private-lb.dns_name}:8081
```

Comment:
Should we make the load balancer port different than the backend port?
443 -> 8081 ?

Reply:
Done